CoPla/.htaccess

```diff
@@ -0,0 +1,19 @@
+AddType application/rdf+xml .rdf
+
+# Rewrite engine setup
+RewriteEngine On
+			
+# Rewrite rule to serve HTML content
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^ontology\$ https://raw.githubusercontent.com/kai-vu/companion-planting-decision-support/refs/heads/main/owl/companion_planting_v6.owl [R=303,L]
+
+# Rewrite rule to serve RDF/XML content if requested
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^ontology\$ ontology.owl [R=303]
```

Comment:
you need to redirect to a URL. `ontology.owl` is not available here. Like you did in the redirection above.

Reply:
Thank you for the pointer! I'm fixing also the documentation right now, so will update this as well.